tao_compiler/mlir/xla/ral/context/stream_executor_based_impl.cc

```diff
@@ -1288,6 +1288,19 @@ bool PickBestAlgorithm(CudnnConvParams& params,
   return true;
 }  // namespace gpu_conv_impl
 
+static bool layout_match(const std::vector<int32_t>& ref,
+                         MemRefType<int32_t, 1> metadata) {
```

Comment:
nit: better to pass `metadata` by const ref, too? It has at least 5 64-bit member inside.

Reply:
Done, thanks.